### DISCUSSION TITLE
Feature Request: Customize Filename Format for TV Show Trailers

### DISCUSSION THREAD

**Discussion by alexkahler** [nandyalu/trailarr#55] [Ideas]
I believe Trailarr could benefit from an option to customize filenames specifically for TV show trailers. TV shows often have multiple seasons, each with its own trailer. Including "Season" in the filename would help keep the files organized and easily identifiable.

To achieve this, it would be useful to introduce a {season_number} tag in the settings, similar to how {title}, {year}, and {resolution} are handled. Additionally, to keep things flexible, there could be separate "Trailer File Name" text boxes in the settings—one for movies and another for TV shows—allowing users to customize the naming conventions based on content type.

This feature would improve file organization, especially for users with large libraries of multi-season TV shows, making it easier to identify trailers and their respective seasons at a glance.

**Comment by nandyalu** [comment]
I am considering something similar to the proposed option. But instead of just adding filename option, I am considering adding Profiles where you can select most of the Trailer options in Settings to a Profile, and assign Profiles to individual Radarr and Sonarr Connections.

**Comment by nandyalu** [comment]
This has been implemented in `v0.4.0`
